ts/components/StatusMessages.tsx

```diff
@@ -1,17 +1,14 @@
-import {
-  AlertEdgeToEdgeProps,
-  AlertEdgeToEdgeWrapper
-} from "@pagopa/io-app-design-system";
+import { AlertEdgeToEdgeWrapper } from "@pagopa/io-app-design-system";
 import { PropsWithChildren } from "react";
 import { useStatusAlertProps } from "../hooks/useStatusAlertProps";
 
 type StatusMessagesProps = PropsWithChildren;
 
 export const StatusMessages = ({ children }: StatusMessagesProps) => {
-  const alertProps: AlertEdgeToEdgeProps | undefined = useStatusAlertProps();
+  const statusAlert = useStatusAlertProps();
 
   return (
-    <AlertEdgeToEdgeWrapper alertProps={alertProps}>
+    <AlertEdgeToEdgeWrapper alertProps={statusAlert?.alertProps}>
       {children}
```

Comment:
The BS is not working. I think we should add the `bottomSheet` component here, under the children.

Reply:
Woops I missed it during a refactoring, addressed in [2c72767](https://github.com/pagopa/io-app/pull/6891/commits/2c7276725c05095119931bdf6c34913125d93e67)